src/copy-filter.js

```diff
@@ -8,6 +8,8 @@ const prune = require('./prune')
 const targets = require('./targets')
 
 const DEFAULT_IGNORES = [
+  'package-lock\\.json$',
+  'yarn\\.lock$',
```

Comment:
One small suggestion:

```suggestion
  '\bpackage-lock\\.json$',
  '\byarn\\.lock$',
```